arxiv-base/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# arxiv/flask base docker file
+
+FROM centos:centos7
```

Comment:
Consider pinning this to an image sha or internally hosted image for best reproducibility.

Reply:
After discussing in person with @rcplane we came to the consensus that pinning (i.e. maintaining our own copy of centos7 in our registry that we have tested) would be good, once we have a registry set up. I'll added `yum -y update security` to the base to partially compromise between having regular updates and having a stable image - this at least allows us to have a known-working base OS image under our control (in our registry) in case the same version on docker hub disappears at some point.